### ISSUE TITLE
Can't access "new AccessibilityEvent()" with Robolectric 4.12

### ISSUE THREAD

**Issue by utzcoz** [robolectric/robolectric#9045]
### Description

Chromium's Robolectric tests use `new AccessibilityEvent()` to obtain an `AccessibilityEvent` instance, and it works with Robolectric 4.11.x. But when I tried to bump Robolectric to 4.12.1 for it, it failed with the reason it can't access `new AccessiblityEvent()`.

### Steps to Reproduce

See https://chromium-review.googlesource.com/c/chromium/src/+/5419446.

I will add extra tests in Robolectric very soon to test its behavior.

### Robolectric & Android Version

Robolectric 4.12.1.

### Link to a public git repo demonstrating the problem:

Robolectric itself.

**Comment by hoisie** [comment]
Can you print the whole stack trace?

**Comment by hoisie** [comment]
@utzcoz Ah I think this is the commit that probably caused the error:

[c47fa04d3ac9699ff820e2dfc663ff0c8a7675c1](https://github.com/robolectric/robolectric/commit/c47fa04d3ac9699ff820e2dfc663ff0c8a7675c1)

TLDR is that Robolectric was making all constructors public, so the update was to be more consistent with real Android.

**Comment by hoisie** [comment]
For instance, the public AccessibilityNodeInfo () constructor was added in SDK 30. Previously, it was a private constructor. However, using new AccessibilityNodeInfo() used to work in all SDK levels in Robolectric. We don't want to mislead people into thinking that is possible with older SDK levels.

**Comment by hoisie** [comment]
To get around the issue:

1) Set the minSdk to 30 (or whatever it is)

2) Use ReflectionHelpers.callConstructor

You just can't use `new AccessibilityEvent()` on older SDks.
